ui-tests/tests/src/config.ts

```diff
@@ -11,18 +11,26 @@
 import * as dotenv from 'dotenv';
 import * as assert from 'assert';
 
-dotenv.config();
+//dotenv.config();
 
 function ensureEnv(e: string): string {
   const result = process.env[e];
   assert.notStrictEqual(typeof result, 'undefined', `Required ${e} to be set in the environment`);
   return result as string;
 }
 
+let config1;
```

Comment:
Silly comment, but can we use a variable that provides more context than 'config1'? Maybe swapConfig or tmpConfig or something else.

Reply:
I just made few changes for this pls revie again thanks @gibaros